dataland-frontend/tests/e2e/utils/EuTaxonomyFinancialsUpload.ts

```diff
@@ -265,7 +267,7 @@ export function uploadCompanyViaApiAndEuTaxonomyDataForFinancialsViaForm(
           `/companies/${storedCompany.companyId}/frameworks/${DataTypeEnum.EutaxonomyFinancials}/upload`
         );
         beforeFormFill();
-        fillAndValidateEuTaxonomyForFinancialsUploadForm(testData);
+        formFillSteps(testData);
```

Comment:
Now we have three functions we get as parameters in a sequence. We should remove beforeFormFill and afterFormFill and put their content into formFillSteps wherever this function is called.

Reply:
Done.